code/game/objects/structures/pipes/vents/vents.dm

```diff
@@ -180,5 +180,5 @@
 	if(!gas_holder || welded)
 		return FALSE
 	playsound(loc, 'sound/effects/smoke.ogg', 25, 1, 4)
-	gas_holder.set_up(radius, 0, get_turf(src), null, 10 SECONDS)
+	gas_holder.set_up(radius, 0, get_turf(src), null, 10)
```

Comment:
deliberately removed the SECONDS multiplier because it's breaking the maths. Time is ticking down one per second rather than per tick which is why gas is currently lasting near 2 minutes.